resources/js/visual-editor/site-editor/styles/panels/panel-controls.css

```
@@ -0,0 +1,134 @@
+.ap-site-editor__styles-inspector .components-panel__row {
+    align-items: stretch;
+    flex-direction: column;
+    margin: 0;
+    min-height: 0;
+    padding-block: 0.5rem;
+}
+
+.ap-site-editor__styles-inspector
+    .components-panel__row
+    + .components-panel__row {
+    border-top: 1px solid color-mix(in srgb, currentColor 8%, transparent);
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)